app/assets/stylesheets/global.sass

```diff
@@ -2764,3 +2764,32 @@ nav.navbar.navbar-default ul.dropdown-menu.cycle
   font-weight: bold
   text-align: center
   line-height: 20px
+
+.smart-banner
+  color: #fff
+  font-family: "Merriweather"
+  margin-top: 38px
+
+  @media (min-width: $screen-sm-min)
+    margin-top: 0px
+
+  .info
+    padding: 10px
+    background: linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7))
+    background-color: #883DE1
+    height: 175px
+    position: relative
+
+    .icon-close
+      float: left
+      margin-top: 70px
+      margin-left: 20px
+  
+    .app-icon
+      float: left
```

Comment:
I left the button on the middle left because I found a lot of examples that used that position for the button